From 07a544a9b52fa2b3a0947d21581da63357860eb9 Mon Sep 17 00:00:00 2001
From: Jon Turney <jon.turney@dronecode.org.uk>
Date: Sun, 25 Oct 2015 13:15:07 +0000
Subject: [PATCH 08/13] Fix some incorrect QUICKREF lines

makedoc defines a command as 'is all upper case,and alone on a line'.  A few
QUICKREF lines currently violate this by having some additional text after the
QUICKREF.

These lines are treated as an unknown command.

This is benign as QUICKREF currently does nothing but produce some ignored
output on stderr.  I'm not sure what the intent of QUICKREF is.
---
 newlib/libm/math/w_acos.c    | 3 ++-
 newlib/libm/math/w_acosh.c   | 3 ++-
 newlib/libm/math/w_asin.c    | 3 ++-
 newlib/libm/mathfp/e_acosh.c | 3 ++-
 newlib/libm/mathfp/s_acos.c  | 3 ++-
 5 files changed, 10 insertions(+), 5 deletions(-)

diff --git a/newlib/libm/math/w_acos.c b/newlib/libm/math/w_acos.c
index fdf2eb0..87fdfd4 100644
--- a/newlib/libm/math/w_acos.c
+++ b/newlib/libm/math/w_acos.c
@@ -58,7 +58,8 @@ RETURNS
 	You can modify error handling for these functions using <<matherr>>.
 
 
-QUICKREF ANSI SVID POSIX RENTRANT
+QUICKREF
+ ANSI SVID POSIX RENTRANT
  acos	 y,y,y,m
  acosf   n,n,n,m
 
diff --git a/newlib/libm/math/w_acosh.c b/newlib/libm/math/w_acosh.c
index 022c5c6..3243a0a 100644
--- a/newlib/libm/math/w_acosh.c
+++ b/newlib/libm/math/w_acosh.c
@@ -60,7 +60,8 @@ Neither <<acosh>> nor <<acoshf>> are ANSI C.  They are not recommended
 for portable programs.
 
 
-QUICKREF ANSI SVID POSIX RENTRANT
+QUICKREF
+ ANSI SVID POSIX RENTRANT
  acos	 n,n,n,m
  acosf   n,n,n,m
 
diff --git a/newlib/libm/math/w_asin.c b/newlib/libm/math/w_asin.c
index e7884b2..6b4466d 100644
--- a/newlib/libm/math/w_asin.c
+++ b/newlib/libm/math/w_asin.c
@@ -59,7 +59,8 @@ return NaN (not a number), set the global variable <<errno>> to
 
 You can change this error treatment using <<matherr>>.
 
-QUICKREF ANSI SVID POSIX RENTRANT
+QUICKREF
+ ANSI SVID POSIX RENTRANT
  asin	 y,y,y,m
  asinf   n,n,n,m
 
diff --git a/newlib/libm/mathfp/e_acosh.c b/newlib/libm/mathfp/e_acosh.c
index c2bf528..68ea35d 100644
--- a/newlib/libm/mathfp/e_acosh.c
+++ b/newlib/libm/mathfp/e_acosh.c
@@ -49,7 +49,8 @@ Neither <<acosh>> nor <<acoshf>> are ANSI C.  They are not recommended
 for portable programs.
 
 
-QUICKREF ANSI SVID POSIX RENTRANT
+QUICKREF
+ ANSI SVID POSIX RENTRANT
  acos    n,n,n,m
  acosf   n,n,n,m
 
diff --git a/newlib/libm/mathfp/s_acos.c b/newlib/libm/mathfp/s_acos.c
index 88d5937..dcd618f 100644
--- a/newlib/libm/mathfp/s_acos.c
+++ b/newlib/libm/mathfp/s_acos.c
@@ -50,7 +50,8 @@ o $\pi$.
         You can modify error handling for these functions using <<matherr>>.
 
 
-QUICKREF ANSI SVID POSIX RENTRANT
+QUICKREF
+ ANSI SVID POSIX RENTRANT
  acos    y,y,y,m
  acosf   n,n,n,m
 
-- 
2.5.3

